package.json

```diff
@@ -78,7 +79,7 @@
 		"mocha-junit-reporter": "^2.0.0",
 		"nyc": "^15.0.0",
 		"oclif": "^4.0.3",
-		"sinon": "^13",
+		"sinon": "^17",
```

Comment:
My IDE started complaining super loudly about sinon being behind, so I updated it and added the type module.